packages/client/components/ReflectionGroup/DraggableReflectionCard.tsx

```diff
@@ -88,7 +92,7 @@ const DraggableReflectionCard = (props: Props) => {
     staticReflectionCount,
     swipeColumn
   )
-  const isDragPhase = phaseType === 'group' && !isComplete
+  const isDragPhase = phaseType === 'group' && !isComplete && !inSpotlight
```

Comment:
we'll want the cards to be draggable inside the spotlight & if someone else is dragging the card it should be moving around in the spotlight, too

Reply:
Makes sense. Just to make sure we're on the same page with how the spotlight dragging behaviour will work: 

- if the viewer opens the spotlight modal, the spotlight modal only opens for the viewer, not for everyone in the meeting
- if someone that is not the viewer drags the reflection that's in the spotlight, the reflection would drag as normal for the non-viewer
- if someone that is not the viewer drags the reflection that's in the spotlight into a new reflection group, nothing would change in the spotlight for the viewer. If the viewer then drags the spotlight reflection into one of the reflection groups in the spotlight, the reflection would move to the new reflection group
- if the viewer moves the spotlight reflection to a reflection group in the spotlight, non-viewers would see the remote reflection move from it's original position on the kanban to its new destination

Reply:
> if someone that is not the viewer drags the reflection that's in the spotlight into a new reflection group, nothing would change in the spotlight for the viewer.

the opposite for this one. if you've got spotlight open and see the card "Cats", I can drag that around. if i hover a group in your spotlight, it you see that. if i hover a group not in your spotlight, you just see it floating somewhere. if i release the mouse, you see the card go back to its spot. if i drop it on a group in your spotlight, you see that group grow in size.